modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -5,8 +5,12 @@
 [id="virt-removing-interface-from-nodes_{context}"]
 = Removing an interface from nodes
 
-Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and set
-the `state` of the interface to `absent`.
+You can remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and setting the `state` of the interface to `absent`.
+
+Removing an interface from a node does not automatically restore the node network configuration to a previous state. If you want to restore the previous state, you will need to define that node network configuration in the policy.
+
+If you remove a bridge or bonding interface, any of the node NICs throughout the cluster that were previously attached or subordinate to that bridge or bonding interface are placed in a `down` state and become unreachable. +
+To maintain connectivity with the NIC, specify a NIC configuration in the same policy with a status of `up` and either DHCP or a static IP address.
```

Comment:
This sentence is a little unclear to me. Some potentially ill-advised suggestions:

s/with the NIC/with the NIC that you removed (? I'm not sure which NIC you're referring to here.)
s/specify a NIC configuration/configure a NIC
s/with a status/that has a status
s/and either/that uses either

Reply:
NIC here refers to the node NIC that was previously attached or subordinate to the removed bridged/bonded interface. With your feedback in mind I've removed the line break so that it is one paragraph and modified that last sentence:
To avoid losing connectivity, configure the node NIC in the same policy so that it has a status of `up` and either DHCP or a static IP address.